### PR TITLE
rfc/unwind-create

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
This PR adds an initial proposal on using `UNWIND` for creating multiple nodes in the same query. This has readability and performance improvements

### CODE REVIEW

**Review by MacondoExpress — COMMENTED** [review]
It looks good to me! I think there are some edge cases still to consider, for example:

* Relationship properties.
* N-depth creation.

For N-depth creation, let's consider this case:
```graphql
mutation Mutation {
  createMovies(
    input: [
      { title: "The Matrix", actors: { create: [{ node: { 
        name: "Keanu",
        address: { 
          create: { 
	    node: { 
	      city: "Los Angeles", 
	      state: "California", 
	      street: "22, 53th street"
	    }
	  } 
        } 
       } }] } }
      { title: "The Matrix Resurrection" }
    ]
  ) {
    movies {
      title
      actors {
        name
      }
    }
  }
}
```

**Comment by angrykoala** [comment]
Complex examples added, this should be fully pitched now

File: docs/rfcs/rfc-024-unwind-create.md [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+            title
+            actors {
+                name
+            }
+        }
+    }
+}
+```
+
+Currently, the previous query will yield the following Cypher:
```
Comment (mjfwebb) [inline docs/rfcs/rfc-024-unwind-create.md:91]:
```suggestion
The query without `UNWIND` will yield the following Cypher:
```

**Review by mjfwebb — COMMENTED** [review]
Just some suggestions for the wording

File: docs/rfcs/rfc-024-unwind-create.md [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+
+RETURN [
+this0 { .title },
+this1 { .title }] AS data, meta
+
+```
+
+# Proposed solution
+
+Alternatively, `UNWIND` can be used to improve performance for batch create. This performance improvement has already been verified:
```
[the remark this answers is not in the document]
Comment (angrykoala) [reply docs/rfcs/rfc-024-unwind-create.md:58]: :clap:

**Pull request merged**